docs/data_management/tutorial/globus_individual_tutorial.md

```diff
@@ -0,0 +1,367 @@
+        - **(2)** Transferring with a computer on other networks? select "Cheaha cluster **off**-campus (UAB Science DMZ)".
+
+        ![UAB Box search results.](./images/gi-uab-collections/001-cheaha.png)
+
+    - **Long-Term Storage (LTS)**: Type "UAB LTS" and select the entry labeled "UAB Research Computing LTS (Long Term Storage aka S3)".
+
+        ![UAB Box search results.](./images/gi-uab-collections/002-lts.png)
+
+    - **UAB Box**: Type "UAB Box" and select the entry labeled "UAB Box".
+
+        ![UAB Box search results.](./images/gi-uab-collections/003-box.png)
+
+Proceed on to learn [how to find Collections shared with you](#how-do-i-find-uab-storage-collections).
+
+[Return to the top](#globus-tutorials-for-individual-researchers).
+
+## How Do I Find Collections Shared with Me?
+
+Globus allows users to share Collections with others. Other researchers, labs, and Research Cores on campus may invite you to their Collections to share data with you. The following instructions let you view Collections shared with you.
+
+1. [Log in to Globus using your credentials.](#how-do-i-get-onto-the-globus-web-app)
+1. You should see the "File Manager" page. If not click "File Manager" in the left-hand navigation panel.
+
+    ![File Manager page.](./images/common/file-manager.png)
+
+1. Click the "Collection Search" bar to open the "Collection Search" page.
+
+    ![Collection search page](./images/common/collection-search-page.png)
+
+1. The list of Collections will be filtered down to all Collections others have granted you access to, which should help you find the Collections you need.
+
+    ![Collection Search page of Globus Web App. "Shared With You" tab is selected and the list of Collections has been filtered down.](./images/gi-find-shared-collections/003-list.png)
+
+1. Click the name of the Collection to be taken back to the file manager page with the Collection filled in.
+
+    ![File Manager page of the Globus Web App. One of the "Collection Search" bars is filled in with the previously selected Collection.](./images/gi-find-shared-collections/004-selected.png)
+
+If you can't find a particular Collection this way, but know its name, try [searching for Collections](#how-do-i-search-for-collections-by-name).
+
+Proceed on to learn how to transfer between Collections.
+
+- [Between a Collection and Cheaha](#how-do-i-transfer-between-a-collection-and-cheaha)
+- [Between a Collection and LTS](#how-do-i-transfer-between-a-collection-and-lts)
+- [Between LTS and Cheaha](#how-do-i-transfer-between-lts-and-cheaha)
+
+[Return to the top](#globus-tutorials-for-individual-researchers).
+
+## How Do I Transfer between a Collection and Cheaha?
+
+Please use the following instructions to transfer data from a shared Collection to a [bucket on LTS](../lts/index.md). These instructions may also be used for other buckets on LTS, provided you have access and know their file path.
```

Comment:
A typo, I think, the  "[bucket on LTS](../lts/index.md)"  should be "**Cheaha**", as the transfer is between shared collection and Cheaha.